### PR TITLE
edX staff change role assignment email.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6468](https://openedx.atlassian.net/browse/ECOM-6468)

As edX staff when I change the role assignments on a course, I'd like to notify all edX users currently assigned to the Course about the change so that they are aware.

**AC:**

1. Validate that when an edX staff changes role assignments, an email is sent to all edX users currently assigned to the Course

Note:
Email content will be addressed in a separate ticket

### CODE REVIEW

**Comment by waheedahmed** [comment]
@tasawernawaz @awais786 Please review.

**Review by bderusha — APPROVED** [review]

File: course_discovery/apps/publisher/emails.py [hunk 1]
```diff
@@ -449,3 +449,55 @@ def send_course_run_published_email(course_run):
         )
         logger.exception(error_message)
         raise Exception(error_message)
+
+
+def send_change_role_assignment_email(course_role, former_user):
+    """ Send email for role assignment changed.
+
+        Arguments:
+            course_role (Object): CourseUserRole object
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/emails.py:458]: nit: add `former_user` in arguments

**Review by tasawernawaz — APPROVED** [review]
Just a nit, otherwise 👍

File: course_discovery/apps/publisher/emails.py (hunk 1 above)
Comment (waheedahmed) [reply course_discovery/apps/publisher/emails.py:458]: Added.

**Pull request merged**